pyaedt/edb_core/EDB_Data.py

```diff
@@ -13,6 +16,431 @@
     )
 
 
+class EDBNetsData(object):
+    """Manages EDB functionalities for a primitives.
+    It Inherits EDB Object properties.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edb = Edb(myedb, edbversion="2021.2")
+    >>> edb_net = edb.core_nets.nets["GND"]
+    >>> edb_net.name # Class Property
+    >>> edb_net.GetName() # EDB Object Property
+    """
+
+    def __getattr__(self, key):
+        if key in dir(self):
+            try:
+                return getattr(self, key)
+            except:
+                raise AttributeError("Attribute not present")
+
+        if key in dir(self.net_object):
+            try:
+                return getattr(self.net_object, key)
+            except:
+                raise AttributeError("Attribute not present")
+
+    # def __setattr__(self, key, value):
+    #     if key in dir(self):
+    #         try:
+    #             return setattr(self, key, value)
+    #         except:
+    #             raise AttributeError("Attribute not present")
+    #     else:
+    #         try:
+    #             return setattr(self.rimitive, key, value)
+    #         except:
+    #             raise AttributeError("Attribute not present")
```

Comment:
@maxcapodi78 Can it be deleted or otherwise remove the comments?

Reply:
Done. Thanks